consensus/misc/eip1559.go

```diff
@@ -33,11 +35,35 @@ const MaximumL2BaseFee = 10000000000
 // `scalar` accounts for finalization costs. `overhead` accounts for sequencing and proving costs.
 // we use 1e18 for precision to match the contract implementation.
 var (
-	BaseFeePrecision       = new(big.Int).SetUint64(1e18)
-	DefaultBaseFeeScalar   = new(big.Int).SetUint64(34000000000000)
-	DefaultBaseFeeOverhead = new(big.Int).SetUint64(15680000)
+	BaseFeePrecision = new(big.Int).SetUint64(1e18)
+
+	// scalar and overhead are updated automatically in `Blockchain.writeBlockWithState`.
+	baseFeeScalar   = big.NewInt(0)
+	baseFeeOverhead = big.NewInt(0)
```

Comment:
Might an API to check these values be helpful? Another would be to observe the logs during updating params.